ntoskrnl/config/cmnotify.c

```diff
@@ -14,6 +14,77 @@
 
 /* FUNCTIONS *****************************************************************/
 
+PCM_NOTIFY_BLOCK
+NTAPI
+CmpAllocateNotifyBlock(IN CM_NOTIFY_MODE NotifyMode,
+                       IN PCM_KEY_BODY Key,
+                       IN PCM_NOTIFY_BLOCK MasterBlock OPTIONAL)
```

Comment:
For new functions, please use SAL2 annotations. Here's an example:
```suggestion
PCM_NOTIFY_BLOCK
NTAPI
CmpAllocateNotifyBlock(
    _In_ CM_NOTIFY_MODE NotifyMode,
    _In_ IN PCM_KEY_BODY Key,
    _In_opt_ PCM_NOTIFY_BLOCK MasterBlock)
```